frame/democracy/src/benchmarking.rs

```diff
@@ -70,7 +70,7 @@ fn add_referendum<T: Config>(n: u32) -> Result<ReferendumIndex, &'static str> {
 	let referendum_index: ReferendumIndex = ReferendumCount::<T>::get() - 1;
 	T::Scheduler::schedule_named(
 		(DEMOCRACY_ID, referendum_index).encode(),
-		DispatchTime::At(1u32.into()),
+		DispatchTime::At(2u32.into()),
```

Comment:
why this change ?

Reply:
When running benchmarks, we set the block number to 1, versus before in tests, we had block number as 0.

This change avoids an error where we try to schedule an event to happen on the same block.